odo service create from operator hub feature failed on IBM Z and P

/kind feature

<!--

Welcome! - We kindly ask you to:

  1. Fill out the issue template below 
  2. Use the Google group if you have a question rather than a bug or feature request.
  
The group is at: https://groups.google.com/forum/#!forum/odo-users

Thanks for understanding, and for contributing to the project!

-->

## Which functionality do you think we should add?
We are working on odo CI enablement for Z/P. For make test-operator-hub , most of the tests failed because of error exec format error. 
## Why is this needed?
To enable the tests on IBM P/Z, the default images used for operator-hub should support s390x or ppcl64le.

I pasted the log here:
```
oc logs pod/vpkbkk-d74779f5b-j4gpw -n pkarjirekf
time="2020-08-12T10:08:52Z" level=info msg="create process:devrun"
time="2020-08-12T10:08:52Z" level=info msg="create process:debugrun"
time="2020-08-12T10:08:57Z" level=debug msg="no auth required"
time="2020-08-12T10:08:57Z" level=info msg="stop the program" program=devrun
time="2020-08-12T10:08:57Z" level=info msg="stop the program" program=debugrun
time="2020-08-12T10:08:57Z" level=info msg="force to kill the program" program=debugrun
time="2020-08-12T10:08:57Z" level=info msg="force to kill the program" program=devrun
time="2020-08-12T10:08:58Z" level=debug msg="no auth required"
time="2020-08-12T10:08:58Z" level=debug msg="succeed to find process:devrun"
time="2020-08-12T10:08:58Z" level=info msg="try to start program" program=devrun
time="2020-08-12T10:08:58Z" level=info msg="success to start program" program=devrun
ODO_COMMAND_RUN is npm start
Changing directory to ${PROJECTS_ROOT}
Executing command cd ${PROJECTS_ROOT} && npm start
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /projects/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/projects/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /opt/app-root/src/.npm/_logs/2020-08-12T10_08_58_473Z-debug.log
time="2020-08-12T10:08:59Z" level=debug msg="wait program exit" program=devrun
time="2020-08-12T10:08:59Z" level=error msg="program stopped with error:exit status 254" program=devrun
time="2020-08-12T10:08:59Z" level=info msg="Don't start the stopped program because its retry times 0 is greater than start retries 0" program=devrun
```

--- a/scripts/configure-installer-tests-cluster-s390x.sh
+++ b/scripts/configure-installer-tests-cluster-s390x.sh
@@ -23,7 +23,7 @@ IMAGE_TEST_NAMESPACES="openjdk-11-rhel8 nodejs-12-rhel7 nodejs-12 openjdk-11"
 ## Let developer user have access to the project
 ##oc adm policy add-role-to-user edit developer
 
-#sh $SETUP_OPERATORS
+sh $SETUP_OPERATORS
 # OperatorHub setup complete
 
 # Create the namespace for e2e image test apply pull secret to the namespace

--- a/scripts/openshiftci-presubmit-all-tests.sh
+++ b/scripts/openshiftci-presubmit-all-tests.sh
@@ -39,6 +39,7 @@ if [ "${ARCH}" == "s390x" ]; then
     make test-integration-devfile
     make test-cmd-login-logout
     make test-cmd-project
+    make test-operator-hub
     # E2e tests
     make test-e2e-all
 elif  [ "${ARCH}" == "ppc64le" ]; then

